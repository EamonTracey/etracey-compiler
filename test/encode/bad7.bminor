"invalid \0xq hex"
