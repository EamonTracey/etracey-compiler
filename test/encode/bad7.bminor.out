error: hex backslash code code must be a printable character.
